File: replace_phone.js
#!/usr/bin/env node
const fs = require('fs');
const path = require('path');
const glob = require('glob');

const OLD_PHONE = "9493700577";
const NEW_PHONE = "9390370577";

function replaceInFile(filepath) {
  try {
    let content = fs.readFileSync(filepath, 'utf8');
    const originalContent = content;
    
    if (content.includes(OLD_PHONE)) {
      content = content.split(OLD_PHONE).join(NEW_PHONE);
      fs.writeFileSync(filepath, content, 'utf8');
      const count = (originalContent.match(new RegExp(OLD_PHONE, 'g')) || []).length;
      return count;
    }
    return 0;
  } catch (e) {
    console.error(`Error processing ${filepath}: ${e.message}`);
    return -1;
  }
}

const outDir = path.join(__dirname, 'out');
let total = 0;

console.log("Processing out/ directory...");
if (fs.existsSync(outDir)) {
  const files = fs.readdirSync(outDir)
    .filter(f => f.endsWith('.html') || f.endsWith('.txt'))
    .map(f => path.join(outDir, f));
  
  for (const file of files) {
    const count = replaceInFile(file);
    if (count > 0) {
      total += count;
      console.log(`✓ ${path.basename(file)} - ${count} replacement(s)`);
    } else if (count === 0) {
      console.log(`- ${path.basename(file)} - no matches`);
    } else {
      console.log(`! ${path.basename(file)} - error`);
    }
  }
} else {
  console.log("! out/ directory not found");
}

console.log(`\nTotal replacements: ${total}`);
process.exit(0);

File: replace_phone.py
#!/usr/bin/env python3
import os
import glob

# Root directory files
root_files = ["about.html", "paints.html", "pipes.html", "sanitary.html", "service.html"]

# Counter for total replacements
total_replacements = 0

print("Processing root directory HTML files...")
for filename in root_files:
    filepath = os.path.join(".", filename)
    if os.path.exists(filepath):
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        
        original_content = content
        # Replace all occurrences
        content = content.replace("9493700577", "9390370577")
        
        # Count replacements
        count = original_content.count("9493700577")
        
        if count > 0:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)
            total_replacements += count
            print(f"  ✓ {filename} - replaced {count} occurrence(s)")
        else:
            print(f"  - {filename} - no matches found")
    else:
        print(f"  ! {filename} - not found")

# Process out/ directory
out_dir = "./out"
print("\nProcessing out/ directory files...")
if os.path.exists(out_dir):
    for filename in os.listdir(out_dir):
        filepath = os.path.join(out_dir, filename)
        if os.path.isfile(filepath):
            try:
                with open(filepath, 'r', encoding='utf-8') as f:
                    content = f.read()
                
                original_content = content
                # Replace all occurrences
                content = content.replace("9493700577", "9390370577")
                
                # Count replacements
                count = original_content.count("9493700577")
                
                if count > 0:
                    with open(filepath, 'w', encoding='utf-8') as f:
                        f.write(content)
                    total_replacements += count
                    print(f"  ✓ {filename} - replaced {count} occurrence(s)")
            except Exception as e:
                print(f"  ! {filename} - skipped (binary or error)")
else:
    print("  ! out/ directory not found")

print("\n" + "="*50)
print(f"Total phone number replacements: {total_replacements}")
